docs/static/_redirects

```diff
@@ -50,3 +50,8 @@
 /:version/porter-darwin-amd64           https://github.com/getporter/porter/releases/download/:version/porter-darwin-amd64 302
 /:version/porter-linux-amd64            https://github.com/getporter/porter/releases/download/:version/porter-linux-amd64 302
 /:version/porter-windows-amd64.exe      https://github.com/getporter/porter/releases/download/:version/porter-windows-amd64.exe 302
+
+# Redirect the schema json files
+/schema/v1/credential-set.schema.json https://raw.githubusercontent.com/getporter/porter/release/v1/pkg/schema/credential-set.schema.json
```

Comment:
Instead of redirecting each file individually, let's take advantage of the netlify redirect tricks like  * and :splat so that we can just have a single entry that will work for any schema file.

Also you will need to commit the change to the redirect file to the main branch too in order for netlify to use it with porter.sh.